specification/ai/OpenAI.Assistants/tools/tool_resources.tsp

```diff
@@ -160,5 +160,6 @@ model UpdateCodeInterpreterToolResourceOptions {
 model UpdateFileSearchToolResourceOptions {
   /** A list of vector store IDs to override the current list of the assistant. */
   @maxItems(1)
+  @encodedName("application/json", "vector_store_ids")
```

Comment:
I think we need this annotation, don't we?

Reply:
Yeah. We need this annotation.